blazor/diagram/nodes/customization.md

```diff
@@ -554,9 +554,54 @@ The following code shows how to set the AdditionalInfo value.
     }
 }
 ```
-
 **Note:** You can set any type of value for the AdditionalInfo property.
 
+## How to set ZIndex property for node
+
+* The nodes z-order property specifies the stack order of the node. A node with greater stack order is always in front of a node with a lower stack order. By default value is -1.
```

Comment:
* The nodes' z-order property specifies the stack order of the node. A node with a greater stack order is always in front of a node with a lower stack order. By default, the value is -1.